### ISSUE TITLE
SQLite duplicate column name errors

### ISSUE THREAD

**Issue by paulbaird87** [Cloud9Developer/Jellyfin-Newsletter-Plugin#115]
Fresh convert to Jelly after 5 years with Plex. Loving it so far, though finally come across my first hurdle. 

Newsletter installed fine, 
SMTP settings test fine, 
Didn't touch the HTML format, 
Scraper config set to 'Local JF Hosting' with my public address eg: https://jellyfin.myserver.com

Filesystem Scraper seems to work but when I run task Email Newsletter I get thes errors (in image).
SQLite errors saying it has duplicate columns. 
No newsletter email gets sent.

<img width="654" alt="Jellyfin newsletter help" src="https://github.com/Cloud9Developer/Jellyfin-Newsletter-Plugin/assets/70988346/356c5d1d-5e7a-44e8-82f4-68701d1424fc">

Any help would be much appreciated.

**Comment by Cloud9Developer** [comment]
Those WARNINGS are expected, not errors.

**Comment by Cloud9Developer** [comment]
Yeah, those are the cause. looks like it tries scanning, and fails completely on the first few items? My next couple weeks are gonna be busy (going on vacation), so I won't be able to look into it until after. 

If you can get me a snippet of some debug logs (turn on debug mode in the plugin's settings) that'd help for when I get back!

**Comment by PuerNoctis** [comment]
I have exactly the same problem. Is this issue still under investigation, resp. do you need any output/info from me as well?
